Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -7044,6 +7044,13 @@ function revIDEStackFileVersion pStackFile
       end if
 end revIDEStackFileVersion
 
+function revIDEStackFileIsScriptOnly pStackFile
+   open file pStackFile for text read
+   read from file pStackFile for 1 line
+   close file pStackFile
+   return matchText(it, "script \".*\"")
```

Comment:
@peter-b this is the best I can do until [17770](http://quality.livecode.com/show_bug.cgi?id=17770) is fixed because the `^` will find the BOM